precompiles/pallet-xcm/src/lib.rs

```diff
@@ -49,6 +50,19 @@ mod tests;
 pub const MAX_ASSETS_ARRAY_LIMIT: u32 = 2;
 type GetArrayLimit = ConstU32<MAX_ASSETS_ARRAY_LIMIT>;
 
+pub const XCM_SIZE_LIMIT: u32 = 2u32.pow(16);
+type GetXcmSizeLimit = ConstU32<XCM_SIZE_LIMIT>;
+
+#[derive(
+	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug, MaxEncodedLen, TypeInfo,
+)]
+pub enum TransferTypeHelper {
+	Teleport = 0,
+	LocalReserve = 1,
+	DestinationReserve = 2,
+	RemoteReserve = 3,
```

Comment:
```suggestion

```

Reply:
Removed in aea402dd3193edd09242b0be19e07e3da4e07200.